sage.config.js

```diff
@@ -0,0 +1,72 @@
+const bud = require('@roots/bud')
+
+/**
+ * Extensions
+ */
+bud
+  .extend([
+    require('@roots/bud-sass'),
+    require('@roots/bud-eslint').plugin,
+    require('@roots/bud-stylelint').plugin,
+    require('@roots/bud-purgecss').plugin,
+    require('@roots/bud-wordpress-manifests'),
+  ])
+
+/**
+ * Project paths.
+ */
+bud
+  .srcPath('resources/assets')
+  .publicPath(bud.env.get('APP_PUBLIC_PATH'))
+
+/**
+ * Application assets
+ */
+bud
+  .bundle('app', [
+    bud.src('scripts/app.js'),
+    bud.src('styles/app.scss'),
+  ])
+  .bundle('editor', [
+    bud.src('scripts/editor.js'),
+    bud.src('styles/editor.scss'),
+  ])
+  .bundle('customizer', [
+    bud.src('scripts/customizer.js'),
+  ])
+  .copy('{images,fonts}/**/*')
```

Comment:
I like this change in a big way but I need to check that there are results for the glob before passing it to webpack. right now you'll get an error if there is no match. remove this line or add some files to those dirs if you're testing.